.github/workflows/docker-action.yml

```diff
@@ -18,15 +18,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        image:
-          - 'coqorg/coq:dev'
+        coq_version:
+          - '8.20'
```

Comment:
Maybe it's worth it to test both 8.20 and dev?

Reply:
Something is broken on dev, once that's fixed it would be great to test it.

Reply:
Do you need to replace `Declare ML Module "ltac_plugin".` with `Declare ML Module "coq-core.plugins.ltac".` (possibly followed with `Global Set Default Proof Mode "Classic".`)?  Might also be worth reporting a bug about `-noinit` mode, if this breakage is recent (since remaining / reorganization), there have been a bunch of issues recently around finding OCaml libs, and this one may not get fixed if it's not reported.

Reply:
I wonder if there's some backwards incompatibility in naming / if you need rocq-core instead now